Box secret memory (#400)
paritytech/parity-common: parity-crypto/CHANGELOG.md

## [Unreleased]

> - Put `Secret` memory on heap. [#400](https://github.com/paritytech/parity-common/pull/400)

Reviewer: ```suggestion
## [0.6.2] - 2020-06-19
- Put `Secret` memory on heap. [#400](https://github.com/paritytech/parity-common/pull/400)
```

> - Add `copy_from` conversion methods for `Secret`.
> - Deprecate a few `From` implementations in favor of `copy_from`.

Reviewer: ```suggestion
- Add `copy_from_str` conversion methods for `Secret`.
- Deprecate `From<&str>` in favor of `copy_from_str`.
```

## [Unreleased]

> - Put `Secret` memory on heap.

Reviewer: ```suggestion
- Put `Secret` memory on heap. [#400](https://github.com/paritytech/parity-common/pull/400)
- Add `copy_from` conversion methods for `Secret`.
- Deprecate a few `From` implementation in favor of `copy_from`.
```
